wca-integration/src/main/java/eu/itesla_project/wca/WCAImpl.java

```diff
@@ -402,6 +408,8 @@ public WCACluster after(Path workingDir, ExecutionReport report) throws IOExcept
 
                         // write base state
                         new AmplNetworkWriter(network, dataSource, 0, 0, false, mapper, DOMAINS_AMPL_EXPORT_CONFIG).write();
+                        if (config.isExportStates())
+                            new WCAStateExporter(network, workingDir, 0, 0).export();                        
```

Comment:
Add braces to respect java coding guideline even if there is only one statement.